Sources/Validation/Convenience/IPAddress.swift

```diff
@@ -0,0 +1,25 @@
+private let ipv4Pattern = "(([01]?\\d\\d?|2[0-4]\\d|25[0-5])\\.){3}([01]?\\d\\d?|2[0-4]\\d|25[0-5])"
+
+private let ipv6Pattern =
```

Comment:
Not sure if it's possible, but if this could be better split up it'd make it better maintainable.